components/login/login_controller/login_controller.jsx

```diff
@@ -20,6 +20,7 @@ import Constants from 'utils/constants.jsx';
 import messageHtmlToComponent from 'utils/message_html_to_component';
 import * as TextFormatting from 'utils/text_formatting.jsx';
 import * as Utils from 'utils/utils.jsx';
+import {showNotification} from 'utils/notifications.jsx';
```

Comment:
Our regular notification invocations still use `notifyMe` from `Utils` and thus require no changes.